dask/array/rechunk.py

```diff
@@ -219,10 +222,21 @@ def rechunk(x, chunks="auto", threshold=None, block_size_limit=None):
     dimension to attain blocks of a uniform block size
 
     >>> y = x.rechunk({0: -1, 1: 'auto'}, block_size_limit=1e8)
+
+    Splitting evenly into 6 chunks is possible with the following:
+
+    >>> z = x.rechunk(chunks=(1000 // 6, -1), balance=True)
+    >>> z.chunks
+    ((167, 167, 167, 167, 167, 165), (1000,))
+
+    This means there isn't a chunk of size 4, which is what would happen
+    with ``x.rechunk(chunks=(1000 // 6, -1))``.
```

Comment:
I modified this text.  I hope that you don't mind.

Reply:
Yeah, that's fine. Thanks!